### PR TITLE
Create issue_template.md

### PR DESCRIPTION
**Pull request by klizhentas**

### CODE REVIEW

File: issue_template.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+**What happened**:
+
+**What you expected to happen**:
+
+**How to reproduce it (as minimally and precisely as possible)**:
+
+**Environment**:
+- Gravity version (use `gravity version`):
+- OS (e.g. from /etc/os-release):
+- Output from `gravity status`
```
Comment (r0mant) [inline issue_template.md]: Missing `:` at the end.

File: issue_template.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+**Relevant Debug Logs If Applicable**
+
+- Relevant logs 
+- Caution! Be careful with `gravity report` as it could capture system's private infrormation!
```
Comment (r0mant) [inline issue_template.md]: infrormation -> information

File: issue_template.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+**Have a question or install problem?** Please use [Our Forum](https://community.gravitational.com).
+
+Otherwise, tell us:
+
+**What happened**:
+
+**What you expected to happen**:
+
+**How to reproduce it (as minimally and precisely as possible)**:
```
Comment (r0mant) [inline issue_template.md]: TBH, this "as minimally and precisely as possible" phrase always bothered me for whatever reason :) I would just say "Steps to reproduce".

**Comment by wadells** [comment]
The work here appears to be superceded by https://github.com/gravitational/gravity/pull/423 and the `.github` directory present on master https://github.com/gravitational/gravity/tree/master/.github

**Pull request closed without merging**